### ISSUE TITLE
Incorrect assertion in victoria-metrics-alert/server-deployment.yaml

### ISSUE THREAD

**Issue by cwstrommer** [VictoriaMetrics/helm-charts#168]
victoria-metrics-alert has the following assertion in its `server-deployment.yaml` template:

```
{{- if and (.Values.alertmanager.enabled) (eq .Values.server.notifier.alertmanager.url "") -}}
{{- fail "alert manager URL must be specified" -}}
{{- end -}}
```

I believe that this is incorrect - the alertmanager URL is deterministic when alertmanager is enabled, and therefore `.Values.server.notifier.alertmanager.url` must be provided when alertmanager is **not** enabled.

**Comment by tenmozes** [comment]
Hi sorry for late response,
You're right the condition should have next
```
{{- if and (not .Values.alertmanager.enabled) (eq .Values.server.notifier.alertmanager.url "") -}}
{{- fail "alert manager URL must be specified" -}}
{{- end -}}
```

**Comment by tenmozes** [comment]
the issues is fixed in https://github.com/VictoriaMetrics/helm-charts/commit/f9ffc15a46d002a2b663aee8a2f2b5fc66d09dc9
and helm chart for vmalert with version 0.4.0 has it

**Comment by tenmozes** [comment]
close an issue as fix is available since 0.4.0 version
